examples/sandbox/src/App.js

```diff
@@ -85,6 +86,13 @@ if (process.env.REACT_APP_SOURCE === "SITE_SEARCH") {
       process.env.REACT_SITE_SEARCH_ENGINE_KEY || "Z43R5U3HiDsDgpKawZkA",
     documentType: process.env.REACT_SITE_SEARCH_ENGINE_NAME || "national-parks"
   });
+} else if (process.env.REACT_APP_SOURCE === "WORKPLACE_SEARCH") {
+  connector = new WorkplaceSearchAPIConnector({
+    kibanaBase: "https://search-ui-sandbox.kb.us-central1.gcp.cloud.es.io:9243",
+    enterpriseSearchBase: "https://search-ui-sandbox.ent.us-central1.gcp.cloud.es.io",
+    redirectUri: "http://localhost:3000",
+    clientId: "8e495e40fc1e6acf515e557e534de39d4f727f7f60a3afed24a99ce3a6607c1e",
```

Comment:
We should replace these with environment variables like we do for App Search, and possibly not provide defaults. It might not be a good idea to expose actual cloud details here.

Reply:
Added env variables here: https://github.com/elastic/search-ui/pull/650/commits/55f8f89e2f9af6e1fa597646bcb01245ac6eceb2

I opted for keeping the defaults because 1) it'll be much easier to test all the backends by simply changing one `REACT_APP_SOURCE` value; 2) I think these details are kinda similar to the details we have for Site Search and App Search that lead to swiftype.com; 3) worst-case scenario - hackers will get access to a mostly empty deployment with some non-sensitive data.